pysb/tests/test_simulationresult.py

```diff
@@ -37,3 +40,113 @@ def test_simres_dataframe():
 
     assert df2.shape == (len(tspan1) + len(tspan3),
                          len(model.species) + len(model.observables))
+
+
+def test_save_load():
+    tspan = np.linspace(0, 100, 101)
+    model = tyson_oscillator.model
+    sim = ScipyOdeSimulator(model, integrator='lsoda')
+    simres = sim.run(tspan=tspan, param_values={'k6': 1.0})
+
+    sim_rob = ScipyOdeSimulator(robertson.model, integrator='lsoda')
+    simres_rob = sim_rob.run(tspan=tspan)
+
+    # Reset equations from any previous network generation
+    robertson.model.reset_equations()
+    A = robertson.model.monomers['A']
+
+    # NFsim without expressions
+    nfsim1 = BngSimulator(robertson.model)
+    nfres1 = nfsim1.run(n_runs=2, method='nf', tspan=np.linspace(0, 1))
+
+    # NFsim with expressions
+    nfsim2 = BngSimulator(expression_observables.model)
+    nfres2 = nfsim2.run(n_runs=1, method='nf', tspan=np.linspace(0, 100, 11))
+
+    with tempfile.NamedTemporaryFile() as tf:
+        if os.name == 'nt':
```

Comment:
Is there any reason not to close it unconditionally, regardless of OS? Do Mac/Linux have a problem if it's closed?

Reply:
Closing the file straight away on Mac/Linux as well as Windows does not cause a problem, but the file gets closed at the end of the `with` block anyway. I'll remove the `os.name` test anyway for cross-platform consistency.